setup.py

```diff
@@ -221,6 +221,7 @@
             'inspire_migrator = inspirehep.modules.migrator.tasks',
             'inspire_records = inspirehep.modules.records.tasks',
             'inspire_refextract = inspirehep.modules.refextract.tasks',
+            'inspire_orcid = inspirehep.modules.orcid.tasks',
```

Comment:
Let's keep these in alphabetical order! This should go just after `inspire_migrator`.